APIs/SinkMetadataProcessingAPI.raml

```diff
@@ -117,22 +125,40 @@ mediaType: application/json
           type: !include schemas/resource-list.json
   /{sinkId}:
     get:
-      description: Return Sink.
+      description: List all the API endpoints for a single Sink.
       responses:
         200:
           description: Returned when success.
           body:
             example: !include ../examples/sink-get-200.json
-            type: !include schemas/sink.json
-    /edid:
+            type: !include schemas/sink-api-sink.json
```

Comment:
FYI, @N-Nagorny, several other specs call these schemas like `base`, e.g. `schemas/sink-base.json` in this case.